stream-chat-android-ui-components/src/main/res/layout/stream_ui_messages_header_view.xml

```diff
@@ -48,13 +48,15 @@
 
     <TextView
         android:id="@+id/title"
-        android:layout_width="wrap_content"
+        android:layout_width="0dp"
         android:layout_height="wrap_content"
         android:layout_marginTop="8dp"
         android:lineSpacingExtra="5sp"
+        android:gravity="center_horizontal"
```

Comment:
Will `android:letterSpacing="0"` with ellipsize work for us?

Reply:
I think we should change `  <style name="StreamUiTextAppearance" parent="TextAppearance.MaterialComponents.Body2" />` to `  <style name="StreamUiTextAppearance" parent="TextAppearance.AppCompat.Body2" />` which doesn't apply `letterSpacing` to fix ellipsize issue in other places